### PR TITLE
Updated Russian translation

### PR DESCRIPTION
**Pull request by TotalCaesar659**

### CODE REVIEW

File: locales/ru_RU/pcsx2_Main.po [hunk 1]
```diff
@@ -2246,11 +2267,20 @@ msgstr ""
...
+msgstr ""
+"VU I bit Hack во избежание постоянной рекомпиляции. Исправляет "
+"тормоза/зависания из-за утечек памяти."
+
+#: pcsx2/gui/Panels/GameFixesPanel.cpp:109
 msgid ""
-"VU I bit Hack avoid constant recompilation (Scarface The World Is Yours)"
+"Try this when \"microVU1: Program cache limit\" is printed in the log.\n"
+"Known to affect following games:\n * Scarface The World Is Yours\n * "
+"Crash Tag Team Racing"
```
Comment (lightningterror) [inline locales/ru_RU/pcsx2_Main.po]: Looks like I forgot to revert the change when I removed the gamefix a few days ago so you don't need to include Crash Tag Team Racing.

File: locales/ru_RU/pcsx2_Main.po [hunk 2]
```diff
@@ -2246,11 +2267,20 @@ msgstr ""
...
 msgstr ""
-"VU I bit Hack во избежание постоянной рекомпиляции (Scarface The World Is "
-"Yours)"
+"Включите, если в лог выводится \"microVU1: Program cache limit\".\n"
+"Известно влияние на следующие игры:\n * Scarface The World Is Yours\n * "
+"Crash Tag Team Racing"
```
Comment (lightningterror) [inline locales/ru_RU/pcsx2_Main.po]: Ditto.

**Comment by TotalCaesar659** [comment]
No, I'm not so good in using git. Ask moderators, maybe they can do something.

File: locales/ru_RU/pcsx2_Main.po [hunk 3]
```diff
@@ -1618,6 +1622,14 @@ msgstr "Сбросить все настройки..."
 msgid "Clears all %s settings and re-runs the startup wizard."
 msgstr "Сбрасывает все настройки %s и перезапускает мастер настройки."
 
+#: pcsx2/gui/MainFrame.cpp:520
+msgid "Create &Blockdump"
+msgstr "Создать Block-дамп"
```
Comment (lightningterror) [inline locales/ru_RU/pcsx2_Main.po]: Are you sure this is correct ? I don't see the translation on the menu.

File: locales/ru_RU/pcsx2_Main.po [hunk 4]
```diff
@@ -1618,6 +1622,14 @@ msgstr "Сбросить все настройки..."
...
+
+#: pcsx2/gui/MainFrame.cpp:520
+msgid "Creates a block dump for debugging purposes."
+msgstr "Создает Block-дамп для отладки."
```
Comment (lightningterror) [inline locales/ru_RU/pcsx2_Main.po]: Ditto.

File: locales/ru_RU/pcsx2_Main.po [hunk 5]
```diff
@@ -2074,6 +2083,18 @@ msgstr "Стандартное (4:3)"
...
+msgstr "Отключено"
+
+#: pcsx2/gui/Panels/GSWindowPanel.cpp:39
+msgid "Standard"
+msgstr "Стандартное"
+
+#: pcsx2/gui/Panels/GSWindowPanel.cpp:40
+msgid "Adaptive"
+msgstr "Адаптивное"
+
```
Comment (lightningterror) [inline locales/ru_RU/pcsx2_Main.po]: Same for these 3.

File: locales/ru_RU/pcsx2_Main.po [hunk 6]
```diff
@@ -1528,12 +1534,10 @@ msgstr ""
...
 
 #: pcsx2/gui/MainFrame.cpp:460
-#, fuzzy
 msgid "ISO &Selector"
 msgstr "Выбор ISO"
 
 #: pcsx2/gui/MainFrame.cpp:461
-#, fuzzy
 msgid "Plugin &Menu"
 msgstr "CDVD-плагин"
```
Comment (MrCK1) [inline locales/ru_RU/pcsx2_Main.po]: ISO Selector and Plugin Menu don't seem to work either. Sorry :P

File: locales/ru_RU/pcsx2_Main.po [hunk 7]
```diff
@@ -1680,12 +1691,10 @@ msgid "Boot CDVD (&fast)"
...
 #: pcsx2/gui/MainFrame.cpp:687
-#, fuzzy
 msgid "Boo&t ISO (full)"
 msgstr "Запустить CD/DVD (все этапы)"
 
 #: pcsx2/gui/MainFrame.cpp:688
-#, fuzzy
 msgid "Boot the VM using the current ISO source media"
 msgstr ""
 "Запускает виртуальную машину PS2, используя выбранный DVD-диск или ISO-файл."
```
Comment (MrCK1) [inline locales/ru_RU/pcsx2_Main.po]: These three (Boot ISO (full), Boot ISO(fast), and the VM tooltip from above don't appear translated either.

**Comment by turtleli** [comment]
@lightningterror @MrCK1 The .mo files haven't been updated in the PR (which is fine), that's why the translations aren't appearing.

**Comment by lightningterror** [comment]
Ah so that's why. Not too familiar with how they work so didn't know.
I guess this can be merged then if it's fine.

@refractionpcsx2 leaving it to you to do a Squash Merge.

**Comment by MrCK1** [comment]
Wiki page has been updated.
